### DISCUSSION TITLE
Any news on a fully managed cloud solution?

### DISCUSSION THREAD

**Discussion by PhilParisot** [requarks/wiki#4950] [Help / Questions] [answered]
Our small organization is looking for a wiki we could use internally, we'd like to support an open source project but we do not have the time or expertise to host a solution ourselves.

We wanted to pay for a fully managed solution that was also open-source.

Wiki.js fits this description nicely but I do not see a managed solution, all I could find was this outdated page: https://js.wiki/cloud

Let us know, thanks!

**Comment by Phil-hostwiki** [comment, marked as answer]
There are alternate managed Wiki.js cloud hosting options you can try out.

Requarks is yet to launch theirs. Nico’s reply can be found here
https://twitter.com/requarks/status/1424951142857986074
